src/com/game/client/Client.java

```diff
@@ -2,8 +2,10 @@
 
 import com.game.controller.Game;
 
+import java.io.IOException;
+
 public class Client {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws IOException {
```

Comment:
Is there a reason to throw the IOException here? I think it would be more effective to catch the exception in the method.

Reply:
Fixed